Neos.ContentGraph.DoctrineDbalAdapter/src/DoctrineDbalContentGraphProjection.php

```diff
@@ -1111,6 +1115,7 @@ private function copyReferenceRelations(
 
     private function whenDimensionSpacePointWasMoved(DimensionSpacePointWasMoved $event): void
     {
+        $this->dimensionSpacePoints->insertDimensionSpacePoint($event->target);
```

Comment:
That must happen within the transaction I assume!?

Reply:
Good point!

Reply:
I just realize, it probably does not matter at all, I will add it to the transaction but creating the dimensionspacepoint is fine even if this transaction should fail as we know it is a valid DSP either way, it might just be unused for a while.